liferay-content-management/liferay-portal
# LPD-32586 Previews not honouring file view/download permission.

### What is this trying to solve?
https://liferay.atlassian.net/browse/LPD-40815 Previews not honouring file view/download permission.
### How am I fixing it?
Using DLAppLocalService instead of _dlAppService to avoid permission check, that will be later controlled by MediaQueryProviderImpl._getAdaptiveMediaFromConfigurationEntry
### How can you verify that it works?
1. Add a new image and Check the Download permission for Guest users.
2. Uncheck the View permission for Guest users.
3. Add an image fragment to the home screen and configure it to show the image
6. Open the page in an incognito window (not logged in) and check the result:
7. Image is shown
8. Minimize the browser window to a smaller width than the maximum defined in the adaptive media resolutions (you can see it by inspecting the element through the developer tools), so this way is served by AM. 
9. Check the result
